lib/round.js

```diff
@@ -2,7 +2,8 @@ export class RoundData {
   /** @type {Map<string, string>} */
   #knownStrings
 
-  constructor () {
+  constructor (index) {
```

Comment:
Would you mind adding a jsdoc comment to describe the index type (is it BigInt?)?

I find the name index a bit ambiguous, roundIndex would be more descriptive, but it also feels a bit redundant to have "round.roundIndex", so I am fine to keep the current name.